src/coreclr/vm/perfmap.cpp

```diff
@@ -46,7 +46,11 @@ void PerfMap::Initialize()
 
 const char * PerfMap::InternalConstructPath()
 {
+#ifdef HOST_WINDOWS
```

Comment:
There is at least one more change like this with conditional path for Windows / Unix. Can we do this inside of the CLRConfigNoCache::Get instead? Or is there a case when we don't want to use the PAL_getenv argument on Unix?

Reply:
Ok, makes sense.